dockstore-client/src/test/java/io/dockstore/client/cli/AbstractEntryClientTestIT.java

```diff
@@ -128,7 +128,7 @@ public void testAggregateAWSCredentialNoRegion() {
         String credentials  = ResourceHelpers.resourceFilePath("fakeAwsCredentials");
         String config = ResourceHelpers.resourceFilePath("fakeAwsConfig");
         environmentVariables.set("AWS_CONFIG_FILE", config);
-        environmentVariables.set("AWS_CREDENTIAL_PROFILES_FILE", credentials);
+        environmentVariables.set("AWS_SHARED_CREDENTIALS_FILE", credentials);
```

Comment:
It was like this, but all these duplicate changes make it a good candidate for a constant.